.travis.yml

```diff
@@ -20,17 +20,12 @@ matrix:
     - os: linux
       dist: trusty
       sudo: required
-    - os: linux
-      dist: trusty
-      sudo: required
-      env: SWIFT_SNAPSHOT=$SWIFT_4_DEV_SNAPSHOT
-    - os: osx
-      osx_image: xcode8.3
-      sudo: required
     - os: osx
       osx_image: xcode9
       sudo: required
-      env: SWIFT_SNAPSHOT=$SWIFT_4_DEV_SNAPSHOT
+
+before_install:
+  - git clone https://github.com/IBM-Swift/Package-Builder.git
 
 script:
-  - ./build.sh
+  - ./Package-Builder/build-package.sh -projectDir $TRAVIS_BUILD_DIR
```

Comment:
I understand the only difference between `build.sh` and invoking `build-package.sh` directly was in the Docker case, and we've already removed the docker build, so this seems okay.